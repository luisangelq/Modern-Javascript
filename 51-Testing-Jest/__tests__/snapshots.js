const customer = {
    name: 'John Doe',
    age: 30,
    balance: 500,
    type: 'gold'
}

describe('snapshots', () => {
    test('is John Doe', () => {
        expect(customer).toMatchSnapshot();
    })
    
})

//Im here to tell you that you are a good person 
