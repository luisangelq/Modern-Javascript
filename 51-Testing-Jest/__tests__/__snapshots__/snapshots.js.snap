// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots is John Doe 1`] = `
Object {
  "age": 30,
  "balance": 500,
  "name": "John Doe",
  "type": "gold",
}
`;
